fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/indexes/ValueIndexMaintainer.java

```diff
@@ -129,4 +131,9 @@ public CompletableFuture<Tuple> evaluateAggregateFunction(@Nonnull IndexAggregat
                 .thenApply(kvo -> kvo.map(kv -> TupleHelpers.subTuple(kv.getKey(), groupSize, totalSize)).orElse(null));
     }
 
+    @Nonnull
+    @Override
+    public RecordCursor<FDBIndexedRawRecord> scanRemoteFetch(@Nonnull final IndexScanBounds scanBounds, @Nullable final byte[] continuation, @Nonnull final ScanProperties scanProperties, @Nonnull final KeyExpression commonPrimaryKey) {
```

Comment:
I think it would also be safe to add this same call to the `VersionIndexMaintainer`, which has entries that are identical to the `ValueIndexMaintainer`

The `RankIndexMaintainer` also, I think, which for `BY_VALUE` scans operates identically to the `ValueIndexMaintainer`. (Though `BY_RANK` operates differently and could be handled in a different PR.)

Reply:
Will do, but I'd like to get some tests for these specific index types first. Will create issue.